module/VuFind/src/VuFind/View/Helper/Root/ProxyUrl.php

```diff
@@ -50,11 +61,14 @@ class ProxyUrl extends \Laminas\View\Helper\AbstractHelper
     /**
      * Constructor
      *
+     * @param CacheAdapter           $cache  Cache for web service responses
```

Comment:
Yes and no.  I added it first because I initially wanted it to be a required argument, and $config is optional.  Then for some unclear reason I made it optional as well, but that may have been unintentional.  Easier just to require it?

Reply:
Yeah, if it's a required dependency, then making it required makes sense. If it's optional, then I'd move it to the end. :-)

Reply:
Sure I just wasn't sure if it was ok to make it required, since the cache is only used in practice if the web service mode is enabled.  But since the object cache already exists for many other modules, I doubt it adds much overhead to retrieve it here even if it's not used.  Changed to required.

Reply:
And... now I see it was optional for the sake of some tests.  I guess I will change them both to optional and fix the order.